docs/configuration.md

```diff
@@ -629,3 +629,15 @@ window.$docsify = {
   topMargin: 90, // default: 0
 };
 ```
+
+## loadNavbarOnCover
+
```

Comment:
How about naming it `hideNavbar`.   😃 and plz checking if there has exceptions when this element removed also.

Reply:
It’s just whether it’s displayed on the cover, not all pages